source/topologies/rs-to-sharded.txt

```diff
@@ -0,0 +1,115 @@
+.. _c2c-rs-to-sharded:
+
+=======================================
+Sync a Replica Set to a Sharded Cluster
+=======================================
+
+.. default-domain:: mongodb
```

Comment:
FYI we dont need the default-domain directives anymore, these were leftover from the legacy built tools and can be safely removed from all pages.

```suggestion

```

Reply:
Good to know, just deleted from this page.